### PR TITLE
Run Linting in CI

### PR DESCRIPTION
**Pull request by thebeanogamer**

### CODE REVIEW

**Comment by david-cooke** [comment]
Hi!
Thanks for the contribution to RACTF. Linting is already done on our pre commit hook and your commit reduces test coverage, once app restructure is merged I would be happy to reconsider this pull request.

**Review by david-cooke — COMMENTED** [review]
Please fix the test coverage

**Review by david-cooke — CHANGES_REQUESTED** [review]
^

**Review by david-cooke — CHANGES_REQUESTED** [review]
You appear to have a merge conflict, please resolve.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -54,3 +54,6 @@ jobs:
       with:
         name: coverage.xml
         path: ./coverage.xml
+
+    - name: Run Linting
+      run: make lint
```
Comment (jchristgit) [inline .github/workflows/test.yml:59]: - `make lint` runs `flake8` and `isort`.
- `make format` runs `isort` and `black`.
- Shouldn't `make lint` also run `black --check`?
- Do not use `black --check --diff`, as that does not use cache.

**Review by jchristgit — CHANGES_REQUESTED** [review]
Hi!

Thanks for the contribution to RACTF. Linting is already done on our pre commit hook and your commit reduces test coverage, once app restructure is merged I would be happy to reconsider this pull request.

**Pull request closed without merging**